src/components/home/style.scss

```diff
@@ -0,0 +1,58 @@
+.navbarpp {
+  height: 100px;
+  background-color: #05225a;
+  display: flex;
+  justify-content: space-around;
+  color: white;
+}
+.navBar {
+  list-style: none;
+}
+footer {
+  padding: 70px 150px;
+  position: relative;
+  justify-content: space-around;
+  text-align: center;
+  color: white;
+  background-color: #05225a;
+}
+.icons {
+  svg {
+    margin-bottom: 5px;
+    margin: 15px;
+  }
+}
+.navBar {
```

Comment:
how come the thing in the footer is called the navbar?

* you will want to nest your rules when you can
* don't use `footer` tag, give it a class (it is too general, so it is better when we have somtehing specific. imagine if your teammate wants to use another <footer> tag in the app)`
* add a space between each rule